app/src/main/java/com/example/cocktails/ScrollingActivity.kt

```diff
@@ -33,7 +34,9 @@ import kotlinx.android.synthetic.main.filter_dialog.view.*
 
 
 @Parcelize
-data class Cocktail(val name: String, val type: String, val steps: Array<String>, val ingredients: Array<String>, val image: String, val isCustom: Boolean = false): Parcelable
+data class Cocktail(val name: String, val type: String, val steps: Array<String>,
+                    val ingredients: Array<String>, val image: String, val isCustom: Boolean = false,
+                    val glass: Uri = Uri.parse("Water Glass.sfb")): Parcelable
```

Comment:
@shahafhermann passing a URI object triggers null pointer exception. consider passing a string instead? I just pushed a HF (hot-fix) for this issue, and added (temporary of course) this URI to the AR class.

Reply:
I've actually tested this so it's very weird. I'll try to figure out why this happens, and if I can't make it work - than yes, we should convert this to String.